src/shared/components/create-case-filters/create-case-filters.component.spec.ts

```diff
@@ -618,310 +570,26 @@ describe('CreateCaseFiltersComponent', () => {
-
-    component.filterEventControl.setValue(EVENT_ID_3);
-    component.onEventIdChange();
-    fixture.detectChanges();
-
-    errorElement = de.query($ERROR_SUMMARY);
-    expect(errorElement).toBeFalsy();
-  });
-
-  it('should change button label when callback warnings notified', () => {
-    let callbackErrorsContext: CallbackErrorsContext = new CallbackErrorsContext();
-    callbackErrorsContext.trigger_text = CreateCaseFiltersComponent.TRIGGER_TEXT_START;
-    component.callbackErrorsNotify(callbackErrorsContext);
-
-    fixture.detectChanges();
-    let button = de.query($SELECT_BUTTON);
-    expect(button.nativeElement.textContent).toEqual(CreateCaseFiltersComponent.TRIGGER_TEXT_START);
-
-    callbackErrorsContext.trigger_text = CreateCaseFiltersComponent.TRIGGER_TEXT_CONTINUE;
-    component.callbackErrorsNotify(callbackErrorsContext);
-
-    fixture.detectChanges();
-    expect(button.nativeElement.textContent).toEqual(CreateCaseFiltersComponent.TRIGGER_TEXT_CONTINUE);
-  });
-
-  it('should clear errors and warnings', () => {
-    let callbackErrorsContext: CallbackErrorsContext = new CallbackErrorsContext();
-    callbackErrorsContext.trigger_text = CaseViewerComponent.TRIGGER_TEXT_START;
-    component.callbackErrorsNotify(callbackErrorsContext);
-
-    fixture.detectChanges();
-    component.resetErrors();
-
-    let error = de.query($ERROR_SUMMARY);
-    expect(error).toBeFalsy();
-    expect(component.error).toBeFalsy();
-    expect(component.ignoreWarning).toBeFalsy();
-  });
-
-  it('should disable button when form valid but callback validation errors exists', () => {
-    component.selected.jurisdiction = JURISDICTION_2;
-    component.selected.caseType = CASE_TYPES_2[2];
-    component.selected.event = CASE_TYPE.events[1];
-    component.error = HttpError.from({});
-    fixture.detectChanges();
-
-    let button = de.query($SUBMIT_BUTTON);
-    expect(attr(button, 'disabled')).toEqual(null);
-    const FIELD_ERRORS = [
-      {
-        x: ''
-      }
-    ];
-    const VALID_ERROR = {
-      details: {
-        field_errors: FIELD_ERRORS
-      }
-    };
-    let error = HttpError.from(VALID_ERROR);
-    component.error = error;
-    fixture.detectChanges();
-
-    expect(attr(button, 'disabled')).toEqual('');
-  });
-
-  it('should disable button when form valid but callback errors exist', () => {
-    component.selected.jurisdiction = JURISDICTION_2;
-    component.selected.caseType = CASE_TYPES_2[2];
-    component.selected.event = CASE_TYPE.events[1];
-    fixture.detectChanges();
-
-    let button = de.query($SUBMIT_BUTTON);
-
-    expect(attr(button, 'disabled')).toEqual(null);
-    let error = HttpError.from({});
-    error.callbackErrors = ['anErrors'];
-    component.error = error;
-    fixture.detectChanges();
-
-    expect(attr(button, 'disabled')).toEqual('');
-  });
-
-  it('should enable button when previously disabled for callback errors but new jurisdiction, case type and event selected', () => {
```

Comment:
I know this test wasn't there but wouldn't it be good to add a test that enables and disables the Start button based on whether all dropdowns are selected/some are still to be selected? If there is such tests then fine.

Reply:
What about these two:

  `it('should have an Go button disabled when event is not set', async(() => {
    component.selected.jurisdiction = JURISDICTION_2;
    component.selected.caseType = CASE_TYPES_2[2];
    component.selectedCaseTypeEvents = CASE_TYPE.events;
    component.selected.event = null;
    component.filterEventControl.setValue('');

    component.onEventIdChange();

    fixture.detectChanges();
    let button = de.query($SELECT_BUTTON);
    expect(button.nativeElement.disabled).toBeTruthy();
  }));`

  `it('should have an Go button enabled when event is set', async(() => {
    component.selected.jurisdiction = JURISDICTION_2;
    component.selected.caseType = CASE_TYPES_2[2];
    component.selectedCaseTypeEvents = CASE_TYPE.events;
    component.selected.event = CASE_TYPE.events[1];
    component.filterEventControl.setValue(EVENT_ID_2);

    component.onEventIdChange();

    fixture.detectChanges();
    let button = de.query($SELECT_BUTTON);
    expect(button.nativeElement.disabled).toBeFalsy();
  }));`

Reply:
Sorry, somehow missed those 🤦‍♂️ 😄